File: pinyougou-seckill-interface/src/main/java/com/pinyougou/seckill/service/SeckillGoodsService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

/**
 * 描述
 *
 * @author 三国的包子
 * @version 1.0
 * @package com.pinyougou.seckill.service *
 * @since 1.0
 */
public interface SeckillGoodsService {

    public List<TbSeckillGoods> findAll();


    public TbSeckillGoods findOne(Long id);
}

File: pinyougou-seckill-web/src/main/webapp/js/service/seckillGoodsService.js
app.service('seckillGoodsService',function ($http) {
    this.findAll=function () {
        return $http.get('/seckillGoods/findAll.do');
    }

    this.findOne=function (id) {
        return $http.get('/seckillGoods/findOne.do?id='+id);
    }

    this.submitOrder=function (id) {
        return $http.get('/seckillOrder/submitOrder.do?id='+id);
    }


})